plugins/pastefromlibreoffice/plugin.js

```diff
@@ -14,10 +14,16 @@
 		requires: 'pastetools',
 
 		isSupportedEnvironment: function() {
-			return !CKEDITOR.env.ie || CKEDITOR.env.version >= 11;
+			var isOldIE = CKEDITOR.env.ie || CKEDITOR.env.version < 11;
+
+			return !isSafari() && !isOldIE;
 		},
 
 		init: function( editor ) {
+			if ( isSafari() ) {
```

Comment:
Maybe let's use `isSupportedEnvironment` here? The issue was only about Safari, but it looks inconsistent as we point that older IEs are not supported in `isSupportedEnvironment` but run the plugin code for these browsers too.